modules/apps/object/object-test/src/testFunctional/macros/ObjectField.macro

```diff
@@ -184,6 +190,13 @@ definition {
 			value1 = "${newValue}");
 	}
 
+	macro viewCountCharactersOnEntries {
+		AssertElementPresent(
+			key_fieldName = "${fieldName}",
+			key_viewCountCharacters = "${viewCountCharacters}",
+			locator1 = "ObjectField#VIEW_COUNT_CHARACTERS_ON_ENTRIES");
+	}
+
```

Comment:
@andreefarias it looks like this was caught by SF and added to this commit. In the future, please fix the original commit instead of committing the SF changes. I'll be enforcing this in the future, once we start refactoring.